eyed3 script is launched always with system python, in order
to not being overriden by local python installations or virtualenv.

This conforms to Debian python policy, 2.4.2.
--- a/bin/eyeD3
+++ b/bin/eyeD3
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-python -m eyed3.main "$@"
+/usr/bin/python -m eyed3.main "$@"
